__tests__/homepage.test.tsx

```diff
@@ -17,4 +17,15 @@ describe("Home", () => {
     });
     expect(header).toBeInTheDocument();
   });
+
+  it("renders the expected text content", () => {
+    const { getByText } = render(<Home />);
```

Comment:
Best I could come up with for unit tests on components that don't actually have any content, but is there a more thorough way to do this?

Reply:
This is okay! We also had a hard time testing this on the DS. The accessibility test covers what we want as well.